src/index.tsx

```diff
@@ -64,9 +103,13 @@ export const ErrorBoundaryContext: FC = ({ children }) => {
     <errorBoundaryContext.Provider value={ctx}>
       <ErrorBoundary
         error={error}
-        onError={(...args) => {
-          setError(true);
-          componentDidCatch.current?.(...args);
+        onError={(error: Error | WrappedError, errorInfo) => {
```

Comment:
```suggestion
        onError={(error: Error, errorInfo) => {
```
Could you add the type annotation for `errorInfo`?

Reply:
This is already covered in `ErrorBoundaryProps` at the top.

Reply:
The type here should be unknown right?